cmd/ipfs/daemon.go

```diff
@@ -274,6 +274,7 @@ func daemonFunc(req cmds.Request, res cmds.Response) {
 		}
 	}
 
+	fmt.Printf("Daemon is ready\n")
```

Comment:
The message will surely print last among the other server messages.

However, each of the messages, e.g. `fmt.Printf("Gateway (writable) server listening on %s\n", gatewayMaddr)` prints before each server runs.